core/templates/dev/head/components/forms/schema-based-editors/schema-based-unicode-editor.directive.ts

```diff
@@ -16,6 +16,8 @@
  * @fileoverview Directive for a schema-based editor for unicode strings.
  */
 
+require(
+  'components/forms/custom-forms-directives/apply-validation.directive.ts');
```

Comment:
Should there be empty line after this line? Because `apply-validation` is used in the HTML?